test/simple-mail-forwarder.bats

```diff
@@ -62,6 +62,43 @@
     [ "`cat /etc/mailname`" = "`cat /etc/hostname`" ]
 }
 
+@test "if 2 certs, confirm both have the same domain(s) & CN" {
+    # If there are two certs present make sure they have the same domain(s) & CN
+    if [[ -f /etc/postfix/cert/smtp.ec.cert && -f /etc/postfix/cert/smtp.cert ]]; then
+        ec_cert_subject=`openssl x509 -noout -subject -in /etc/postfix/cert/smtp.ec.cert`
+        rsa_cert_subject=`openssl x509 -noout -subject -in /etc/postfix/cert/smtp.cert`
+        ec_cert_cn=`perl -e 'print join "\n", @cn = $ARGV[0] =~ /(?<=CN\s=\s).*/g;' "$ec_cert_subject" | sort -`
```

Comment:
I used perl here because busy-box grep doesn't include PCRE regex and I needed Lookaround constructors to extract the domains/CN